tools/get_kubectl_kind.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euo pipefail
+source build.env
+
+# This script downloads kind binary that are
+# used as part of the vtop test environment,
+# and places them in vitess/bin/.
+#
+# The upgrade test framework expects kind binary to be found in the PATH.
```

Comment:
There isn't an upgrade test, is there? Maybe this comment just needs to be corrected..

Reply:
Oh yes, that should be vtop example test, I'll fix it

Reply:
Fixed